Documentation/writing-tizen-csharp-apps.md

```diff
@@ -0,0 +1,131 @@
+# Writing a Tizen .NET Application
+
+This document provides you how to write a Tizen .NET Application with Machine Learning APIs.
+Since Tizen 5.5 (not released yet), Machine Learning Inference functionality has been provided on Mobile, Wearable and TV profile.
+
+## Installing Visual Studio Tools for Tizen
+In order to use this functionality, you need to install Visual Studio Tools for Tizen and Tizen SDK version 5.5 or higher. You can find the detailed guide for this at the below link.
+
+* https://developer.tizen.org/development/visual-studio-tools-tizen/installing-visual-studio-tools-tizen
+
+
+# Machine Learning Inference
+
+The Machine Learning Inference features introduce how you can easily invoke the neural network model and get the inference output result.
+
+With Tizen.MachineLearning.Inference.SingleShot, you can load the existing neural network model or your own specific model from storage and invoke it with a single instance of input data, finally get the inference output result.
+
+Currently, this is an experimental functionality and is likely to be changed.
```

Comment:
Is it really experimental?

Reply:
ACK. I will remove an `experimental functionality` and add your sentence.